datafusion/sql/src/expr/value.rs

```diff
@@ -160,9 +161,17 @@ impl<'a, S: ContextProvider> SqlToRel<'a, S> {
         }
     }
 
+    /// Convert a SQL interval expression to a DataFusion logical plan
+    /// expression
+    ///
+    /// Waiting for this issue to be resolved:
+    /// https://github.com/sqlparser-rs/sqlparser-rs/issues/869
+    #[allow(clippy::too_many_arguments)]
```

Comment:
Ignoring clippy for now rather than creating a temporary struct to pass to this function. See: https://github.com/sqlparser-rs/sqlparser-rs/issues/869

Reply:
I can create a pull request on sqlparser if this looks fine.

Reply:
Great job👍 Thank you @aprimadi